### ISSUE TITLE
hwoto add the entry for  new users for helm postgresql-ha

### ISSUE THREAD

**Issue by bocblca** [bitnami/charts#12473]
### Name and Version

bocblca 

### What is the problem this feature will solve?

hwoto add the entry for  new users for helm postgresql-ha

### What is the feature you are proposing to solve the problem?

when I  link  k8s  pod  [postgresql-ha-pgpool] with user postgres
error info
```
psql: error: connection to server at "127.0.0.1", port 5432 failed: FATAL:  SCRAM authentication failed
DETAIL:  pool_passwd file does not contain an entry for "postgres"
db userslist:
postgres=# \du
                                   List of roles
 Role name |                         Attributes                         | Member of
-----------+------------------------------------------------------------+-----------
 blc       | Superuser, Create role, Create DB                          | {}
 post      | Superuser, Create role, Create DB                          | {}
 postgres  | Superuser, Create role, Create DB, Replication, Bypass RLS | {}

 kubectl exec -it rcbpq-ha-postgresql-ha-pgpool-7645fc5cfc-jbzdc  -n net6 -- cat /opt/bitnami/pgpool/conf/pool_passwd
result: 
blc:AES7+5ntWJDYINccEWI8crTYg==
```
my question:
hwoto add the entry for  this user->postgres  for  pgpool?

---thank a lot!

### What alternatives have you considered?

hwoto add the entry for  new users for helm postgresql-ha

**Comment by rafariossaa** [comment]
Hi,
  Have you tried with: `pgpool.usernames` ?

**Comment by smolinari** [comment]
I am also running into a similar problem. 

When I entered `customUsers` `usernames` and `passwords` for the first time, my application, Keycloak, could connect to `pgpool`. I was ecstatic. Then I wanted to move the data to a secret, and after that nothing worked. I couldn't get hard coded credentials anymore either and I tried many, many times. Though, I do believe I got past the username and password being wrong. The last error I had when I stopped trying to solve this was, "the user or password isn't available in the backend", but they were there for sure. The login role in PG was there with the same password. I'm also not sure if it might be the app (Keycloak). 

My fix was to go back to using the root PG user. 

In general, what I'd like to know is, why does pgpool need this password file to begin with to add access for non-root users for apps to the PG cluster? It's a terrible way to go about security, as it means having to sync passwords between pgpool's passwd file and "the backend" i.e. the actual login roles in the database.  

I'm wondering too now thinking about this, do I have to maybe offer up the AES hashed passwords as the password for the app's access? I'll go and try that. Still.... Hmm.... :thinking: I'm so confused right now :zany_face:. 

Scott
